### PR TITLE
Added changes for making helm-chart compatible with CI tag

### PR DESCRIPTION
**Pull request by Jonsy13**
Signed-off-by: Jonsy13 <vedant.shrotria@harness.io>

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] DCO signed
- [X] Chart Version bumped
- [X] Variables are documented in the README.md

### CODE REVIEW

File: charts/litmus/Chart.yaml [hunk 1]
```diff
@@ -23,5 +23,5 @@ icon: https://raw.githubusercontent.com/litmuschaos/icons/master/litmus.png
 dependencies:
 - name: mongodb
   version: "12.1.11"
-  repository: "https://charts.bitnami.com/bitnami"
+  repository: "https://raw.githubusercontent.com/bitnami/charts/archive-full-index/bitnami"
```
Comment (Jasstkn) [inline charts/litmus/Chart.yaml:26]: change repo here as well: https://github.com/litmuschaos/litmus-helm/blob/master/.circleci/ct.yaml#L5

Comment (Jonsy13) [reply charts/litmus/Chart.yaml:26]: Yes this worked, Thanks @Jasstkn 🚀

**Review by ispeakc0de — APPROVED** [review]

File: charts/litmus/ci.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Default values.yml for using chart with "ci" images
```
Comment (Jasstkn) [inline charts/litmus/ci.yml:1]: what do you think about reverting name to values-ci.yaml?

Comment (Jonsy13) [reply charts/litmus/ci.yml:1]: Sure reverting it!

**Review by Jasstkn — APPROVED** [review]

**Review by amityt — APPROVED** [review]

**Pull request merged**